io.weldr.cockpit-composer.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<component type="addon">
+  <id>io.weldr.cockpit-composer</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Composer</name>
+  <summary>
+      Web interface for Composer
```

Comment:
Everything in Cockpit is a Web interface :wink: 

I prefer something along the lines of "Build customized operating system images"